.github/workflows/main.yml

```diff
@@ -0,0 +1,42 @@
+name: Dart
+
+on:
+  push:
+    branches: [ main ]
+  pull_request:
+    branches: [ main ]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v2
+
+      # Install Java
+      - uses: actions/setup-java@v1
+        with:
+          java-version: '12.x'
+
+      # Install flutter
+      - uses: subosito/flutter-action@v1
+        with:
+          flutter-version: '1.22.4'
+
+      - name: Print Flutter/Dart version
+        run: flutter --version
+
+      - name: Install dependencies
+        run: flutter pub get
+
+      # Uncomment this step to verify the use of 'dart format' on each commit.
+      # - name: Verify formatting
+      #   run: dart format --output=none --set-exit-if-changed .
```

Comment:
Not using this for the trial project. We _should_ use it for the real project, I think.